asistente_ladm_col/asistente_ladm_col_plugin.py

```diff
@@ -1224,11 +1225,11 @@ def remove_report_dependency(self):
         Utils.remove_dependency_directory(DEPENDENCY_REPORTS_DIR_NAME)
 
     def show_help(self):
-        self.qgis_utils.show_help()
+        show_plugin_help()
```

Comment:
No need for this `plugin.show_help()`, connect directly to `utils.show_help()`

Reply:
Resolve in commit 66474eb